docker/configs/generators/generators/resources/templates/dot-properties/media-api.properties.template

```diff
@@ -7,6 +7,8 @@ auth.keystore.bucket={{KeyBucket}}
 sns.topic.arn={{SnsTopicArn}}
 cors.allowed.origins={{cors}}
 s3.config.bucket={{ConfigBucket}}
+s3.usagemail.bucket={{UsageMailBucket}}
 persistence.identifier=picdarUrn
 mixpanel.token={{mixpanel_token}}
 es.index.aliases.read=readAlias
+quota.store.key=rcs-quota.json
```

Comment:
vs not having it at all. From the looks of it, this prop was used to identify what S3 key to read; if were reading the latest S3 key now we should be able to get rid of this prop. (Though I could be reading the code incorrectly... 😉 )